<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TempControl</class>
 <widget class="QDialog" name="TempControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>299</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Temperature Control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/image/EZCAP_Icon.ico</normaloff>:/image/EZCAP_Icon.ico</iconset>
  </property>
  <widget class="QRadioButton" name="rBtn_autoControl">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>202</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Auto Control</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBtn_TECOFF">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>202</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TEC OFF</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_power">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="vSlider_power_tempControl">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>20</y>
     <width>21</width>
     <height>180</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
   <property name="sliderPosition">
    <number>25</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBtn_manuel">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>202</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>0</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Temp</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_image_tempControl">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>18</y>
     <width>320</width>
     <height>180</height>
    </rect>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="status_tempcontrol" native="true">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>229</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QLabel" name="tempStatus_tempControl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>3</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Temp</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PressStatus_tempControl">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>2</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Press</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PWMStatus_tempControl">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>4</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>PWM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line1_tempcontrol">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>2</y>
      <width>3</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(48, 48, 48, 255), stop:0.166 rgba(48, 48, 48, 255), stop:0.333 rgba(0, 0, 0, 255), stop:0.5 rgba(0, 0, 0, 255), stop:0.51 rgba(83, 83, 83, 255), stop:1 rgba(83, 83, 83, 255));</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line2_tempcontrol">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>2</y>
      <width>3</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(48, 48, 48, 255), stop:0.166 rgba(48, 48, 48, 255), stop:0.333 rgba(0, 0, 0, 255), stop:0.5 rgba(0, 0, 0, 255), stop:0.51 rgba(83, 83, 83, 255), stop:1 rgba(83, 83, 83, 255));</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line3_tempcontrol">
    <property name="geometry">
     <rect>
      <x>273</x>
      <y>0</y>
      <width>3</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(48, 48, 48, 255), stop:0.166 rgba(48, 48, 48, 255), stop:0.333 rgba(0, 0, 0, 255), stop:0.5 rgba(0, 0, 0, 255), stop:0.51 rgba(83, 83, 83, 255), stop:1 rgba(83, 83, 83, 255));</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="HumidityStatus_tempControl">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>3</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidity</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_P">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>262</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P参数</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_P">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>262</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_I">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>262</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>I参数</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_I">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>262</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>240.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_D">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>262</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>D参数</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_D">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>262</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>30.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_power_tempControl">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>202</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>9.8%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp_tempControl">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>202</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>T</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="vSlider_Humidity_tempCotrol">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>20</y>
     <width>21</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_Humidity">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>RH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Humidity_tempControl">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>203</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="vSlider_temp_tempCotrol">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>20</y>
     <width>21</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>90</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="vSlider_Press_tempCotrol">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>20</y>
     <width>21</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_Press_tempControl">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>203</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Press</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Press">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>0</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Press</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
